moabb/tests/datasets.py

```diff
@@ -187,17 +187,18 @@ def test_datasets_init(self):
             kwargs = {}
             if inspect.signature(ds).parameters.get("accept"):
                 kwargs["accept"] = True
-            with self.assertLogs(logger="moabb.datasets.base", level="WARNING") as cm:
+            with self.assertLogs(logger="moabb.datasets.base", level="WARNING"):
                 # We test if the is_abrev does not throw a warning.
                 # Trick needed because assertNoLogs only inrtoduced in python 3.10:
                 logger.warning(f"Testing {ds.__name__}")
                 obj = ds(**kwargs)
-            self.assertEqual(len(cm.output), 1)
+            # Commented for now, return in next release
+            # self.assertEqual(len(cm.output), 1)
```

Comment:
I think we should keep this test for all the datasets that are not in `depreciated_list, _, _ = zip(*aliases_list)` (It checks that all the codes for new datasets are matching the class name)

Reply:
```suggestion
            if type(obj).__name__ not in deprecated_list:
                self.assertEqual(len(cm.output), 1)
```